mantidproject/mantid
# Muon bug fixes for TFAsymmetry mode

There appeared to be some errors in muon analysis that meant that the TFAsymmetry mode was not being set correctly on Ubuntu (maybe OSX). This ensures that the group workspaces are always stored in the asymmetry table and that it is set correctly.   
**To test:**


things to test (not on Windows):
Load some data into MuonAnalysis (do current run from HIFI it should be long data) 

Then check:
1. Does loading a pair (e.g. HIFI long) not add anything to TFNormalization
2. In multiple fitting, with long loaded change the selection to all groups. Are the norms added to TFNormalization

3. Load the current run ofrom MUSR
4. TFAsymmetry mode should be active in the data analysis tab. The normalisation should be added to the table. 

Fixes #20647 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
